scripts/create_addon

```diff
@@ -93,9 +93,9 @@ function find_addons() {
 
     # abort when nothing found and not embedded
     if [ $? -ne 0 ]; then
-      printf  "$(print_color CLR_ERROR "ERROR: '$1' matches nothing...")\n" ' '>&$SILENT_OUT
-      printf  "for more informations type: ./scripts/create_addon --help\n" ' '>&$SILENT_OUT
-      exit 1
+      echo  "$(print_color CLR_ERROR "ERROR: '$1' matches nothing...")" ' '>&$SILENT_OUT
+      echo  "for more informations type: ./scripts/create_addon --help" ' '>&$SILENT_OUT
```

Comment:
When converting `printf` to `echo` we can lose the `' '` parameter - that's a hangover from the original build stage `printf` (which outputs a variable number of spaces) and is presumably where these other `printf` statements have been copy & pasted from. So...
```
-      echo  "$(print_color CLR_ERROR "ERROR: '$1' matches nothing...")" ' '>&$SILENT_OUT
-      echo  "for more informations type: ./scripts/create_addon --help" ' '>&$SILENT_OUT
+      echo  "$(print_color CLR_ERROR "ERROR: '$1' matches nothing...")" >&$SILENT_OUT
+      echo  "for more informations type: ./scripts/create_addon --help" >&$SILENT_OUT
```